nncf/torch/quantization/default_quantization.py

```diff
@@ -60,6 +68,11 @@
     ],
     QuantizationTrait.OUTPUT_QUANTIZATION_AS_WEIGHTS: [
         operator_metatypes.PTEmbeddingMetatype,
-        operator_metatypes.PTEmbeddingBagMetatype
+        operator_metatypes.PTEmbeddingMetatype.module_metatype,
```

Comment:
You decided not to add module metatype automatically, didn't you?

Reply:
I overthink and decide that explicit typing is better